doc/flame/components.md

```diff
@@ -10,7 +10,7 @@ This diagram might look intimidating, but don't worry, it is not as complex as i
 All components inherit from the abstract class `Component`.
 
 If you want to skip reading about abstract classes you can jump directly to
-[](#positioncomponent).
+[](#PositionComponent).
```

Comment:
`positioncomponent`

Reply:
Interesting, my IDE wanted it to be this way? Does the casing actually matter?

Reply:
Seems to matter indeed, reverted